hack/pin-bundle-images.sh

```diff
@@ -47,7 +47,7 @@ for MOD_PATH in $(go list -mod=readonly -m -json all | jq -r '. | select(.Path |
     if [[ ${LOCAL_REGISTRY} -eq 1 && ( "$GITHUB_USER" != "openstack-k8s-operators" || "$BASE" == "$IMAGEBASE" ) ]]; then
         SHA=$(curl -s ${REPO_CURL_URL}/$BASE-operator-bundle/tags/list | jq -r .tags[] | sort -u | grep $REF)
     else
-        SHA=$(curl -s ${REPO_CURL_URL}/$BASE-operator-bundle/tag/ | jq -r .tags[].name | sort -u | grep $REF)
+        SHA=$(curl -s ${REPO_CURL_URL}/$BASE-operator-bundle/tag/?filter_tag_name=like:$REF | jq -r .tags[].name)
```

Comment:
should we fail before L53 if $SHA is for some reason empty?

Reply:
@stuggi thanks for the suggestion, yes looks like we need an additional check now to confirm if sha is empty, filter_tag_name doesn't return exit_code=1 if tag doesn't exists

~~~
$ curl https://quay.io/api/v1/repository/openstack-k8s-operators/nova-operator-bundle/tag/?filter_tag_name=like:7fdafd5549c5 
{"tags": [], "page": 1, "has_additional": false}
$ echo $?
0

vs earlier

$curl "https://quay.io/api/v1/repository/openstack-k8s-operators/nova-operator-bundle/tag/?limit=100" |  jq -r '.tags[].name' | grep -i 7fdafd5549c5
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 30427  100 30427    0     0  21671      0  0:00:01  0:00:01 --:--:-- 21671

$ echo $?
1
~~~

Reply:
@stuggi added a check to confirm is sha is empty and fail if its empty